portal-impl/src/com/liferay/portal/service/impl/ClassNameLocalServiceImpl.java

```diff
@@ -146,11 +152,23 @@ public void invalidate() {
 		_classNames.clear();
 	}
 
+	private String _getKey(String value) {
+		if (DBPartition.isPartitionEnabled()) {
+			return StringBundler.concat(
+				value, StringPool.AT, DBPartitionUtil.getCurrentCompanyId());
+		}
+
+		return value;
+	}
+
 	private static final Log _log = LogFactoryUtil.getLog(
 		ClassNameLocalServiceImpl.class);
 
 	private static final Map<String, ClassName> _classNames =
 		new ConcurrentHashMap<>();
 	private static final ClassName _nullClassName = new ClassNameImpl();
 
+	@BeanReference(type = CompanyLocalService.class)
+	private CompanyLocalService _companyLocalService;
```

Comment:
@achaparro what's this unused reference for?

Reply:
hey @shuyangzhou, I have just forgot to remove that reference when we removed the company iteration from the service. I will delete it in the next pull request